src/qt/paymentserver.cpp

```diff
@@ -582,22 +582,22 @@ bool PaymentServer::processPaymentRequest(const PaymentRequestPlus& request, Sen
         // Bitcoin amounts are stored as (optional) uint64 in the protobuf messages (see paymentrequest.proto),
         // but CAmount is defined as int64_t. Because of that we need to verify that amounts are in a valid range
         // and no overflow has happened.
-        if (!verifyAmount(sendingTo.second)) {
+        if (!verifyAmount(sendingTo.nAmount)) {
             emit message(tr("Payment request rejected"), tr("Invalid payment request."), CClientUIInterface::MSG_ERROR);
             return false;
         }
 
         // Extract and check amounts
-        CTxOut txOut(sendingTo.second, sendingTo.first);
+        CTxOut txOut(sendingTo.nAmount, sendingTo.scriptPubKey);
```

Comment:
@laanwj Well I think in terms of understanding the code this was an improvement.